cluster/aws/util.sh

```diff
@@ -25,7 +38,9 @@ source "${KUBE_ROOT}/cluster/common.sh"
 ALLOCATE_NODE_CIDRS=true
 
 NODE_INSTANCE_PREFIX="${INSTANCE_PREFIX}-minion"
-ASG_NAME="${NODE_INSTANCE_PREFIX}-group"
+
+# The ASG name must be unique, so we include the zone
```

Comment:
nit: Expand this acronym to "Auto Scaling Group", at least once, upon introduction.